devops/docker/ci/xenial/Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 68DB5E88 \
 RUN add-apt-repository "deb https://repo.sovrin.org/sdk/deb xenial stable" \
     && apt-get update && apt-get install -y \
         libindy-crypto=0.4.5 \
-        libindy=1.6.7 \
+        libindy=1.8.2 \
```

Comment:
I think you have to bump Dockerfile version as well. Otherwise it will be built with an old version from cache.